server/src/main/java/io/littlehorse/server/KafkaStreamsServerImpl.java

```diff
@@ -196,6 +196,7 @@
 import io.littlehorse.server.streamsimpl.taskqueue.PollTaskRequestObserver;
 import io.littlehorse.server.streamsimpl.taskqueue.TaskQueueManager;
 import io.littlehorse.server.streamsimpl.util.GETStreamObserver;
+import io.littlehorse.server.streamsimpl.util.GETStreamObserverNew;
```

Comment:
To refresh my memory, this is basically the same change that I made on my branch, but in a new class so it still works with the Metadata, because we didn't migrate the metadata over yet to the new TagsCache storage mechanism?

Reply:
Yup, yup exactly that